carpark/utils/parse-sqs-event.js

```diff
@@ -18,17 +18,15 @@
     )
   }
 
-  const carId = sqsEvent.Records[0].body
-  const info = carId.match(/([^/]+)\/([^/]+)\/(.+)/)
-  if (!info) {
+  const body = sqsEvent.Records[0].body
+  if (!body) {
     return
   }
-
-  const [, bucketRegion, bucketName, key] = info
+  const { key, region, bucket } = JSON.parse(body)
 
   return {
-    bucketRegion,
-    bucketName,
+    bucketRegion: region,
```

Comment:
discussion: same here... `region` becomes `bucketRegion`... can we normalise to avoid the renames?